### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by Gadjung**
reorganized toolbox, 
added parameter for chance of 2sources in normal room, 
added button to Generate default sector,
added checkbox to hide mineral icon (so one can see sources/room layout better)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,26 +14,53 @@
...
+				<button onclick="fixAll()">Fix All Rooms</button>
+			</div>
+			<div>
+				<div>Walls Chance</div>
+				<input type="range" name="wallChance" value="25" min="0" max="100" step="1" oninput="this.title=this.value + '%'">
+				
+			</div>
+			<div>
+				<div>2 Source chance</div>
+				<input type="range" name="twoSourcesChance" value="25" min="0" max="101" step="1" oninput="this.title=this.value + '%'">
```
Comment (AlinaNova21) [inline public/index.html]: Last I heard, 101 is an odd max for a percentage ;)

File: public/worker.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ self.q = self.Q
 
 addEventListener('message', msg => {
   if (msg.data && msg.data.action == 'generate') {
+    console.log(`${JSON.stringify(msg.data.twoSourcesChance)}`)
+    console.log(`${JSON.stringify(msg.data.wallChance)}`)
```
Comment (AlinaNova21) [inline public/worker.js]: Minor nitpick, extra debug `console.log` calls left in

File: public/index.html (hunk 1 above)
Comment (Gadjung) [reply public/index.html]: fixed to be proper 💯

File: public/worker.js (hunk 2 above)
Comment (Gadjung) [reply public/worker.js]: should be removed

**Review by AlinaNova21 — APPROVED** [review]

**Pull request merged**